sbin/build.sh

```diff
@@ -903,8 +916,12 @@ addJVMVariant(){
 }
 
 addBuildSHA(){ # git SHA of the build repository i.e. openjdk-build
-  local buildSHA=$(git -C ${BUILD_CONFIG[WORKSPACE_DIR]} rev-parse --short HEAD)
-  echo -e BUILD_SOURCE=\"git:$buildSHA\" >> release
+  local buildSHA=$(git -C ${BUILD_CONFIG[WORKSPACE_DIR]} rev-parse --short HEAD 2>/dev/null)
+  if [[ $buildSHA ]]; then
+    echo -e BUILD_SOURCE=\"git:$buildSHA\" >> release
+  else
+    echo "Unable to fetch build SHA, does a work tree exist?..."
```

Comment:
Do you need to crash out at this point or can you continue?

Reply:
I don't believe so as it would mean that GHA is never be able to build. For jenkins builds it should crash before it ever reaches this point as it would mean that something went wrong when fetching the `openjdk-build` repo.

Reply:
OK, thanks